projects/plugins/jetpack/tests/php/test_deprecation.php

```diff
@@ -82,28 +82,6 @@ function provider_deprecated_defined_functions() {
 		);
 	}
 
-	/**
```

Comment:
Removing test. Since `tracks/client` library was moved, it doesn't actually test anything. Here's the commit that [changed the test](https://github.com/Automattic/jetpack/commit/03540a28d9523e8a1fc573f21657c96d8e2df771#diff-6e481d8c767d2b8b1b95901241a41f20312cb0575803c5523b41e9fb7b9a20f2L133).

`set_error_handler( '__return_null' );` is dangerous in tests. This one hides a fatal that would've happened in production if one ever used `jetpack_require_lib( 'tracks/client' );`.

Also, removing the whole require lib code just leaves the assertion part which passes. We can leave that if it's needed.

Reply:
> Here's the commit that [changed the test](https://github.com/Automattic/jetpack/commit/03540a28d9523e8a1fc573f21657c96d8e2df771#diff-6e481d8c767d2b8b1b95901241a41f20312cb0575803c5523b41e9fb7b9a20f2L133).

The test was broken even before that. When the test class was first added in #12857 / #12859 it was already [setting an error handler that would ignore all errors for everything in the entire class](https://github.com/Automattic/jetpack/pull/12857/commits/cfd820a0320cd299c2d15f9762ef9fbaab4ab11a#diff-6e481d8c767d2b8b1b95901241a41f20312cb0575803c5523b41e9fb7b9a20f2R7-R15).

The patch you linked simply relocated the error handler setting to the one call where it was preventing an error rather than applying it to the entire class.